build-go/main.go

```diff
@@ -4,12 +4,13 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"strings"
 
 	"github.com/asraa/slsa-on-github/build-go/pkg"
 )
 
 func usage(p string) {
-	panic(fmt.Sprintf("Usage: %s <config.yml> <env1:val1,env2:val2>\n", p))
+	panic(fmt.Sprintf("Usage: %s <config.yml> <env1:val1,env2:val2> [--dry]\n", p))
```

Comment:
i will refactor this to use the flag library once merged, it's starting to get messy

Reply:
sure. let's discuss this today.

Reply:
this will done in a follow-up PR, as discussed offline.